node/coinstacks/solana/api/src/controller.ts

```diff
@@ -96,4 +96,30 @@ export class Solana implements BaseAPI {
       throw handleError(err)
     }
   }
+
+  /**
+   * Estimate priority fees for a transaction
+   *
+   * @param {SendTxBody} body to account keys
+   *
+   * @returns {Promise<number | undefined>} priority fee estimate
+   */
+  @Response<BadRequestError>(400, 'Bad Request')
+  @Response<ValidationError>(422, 'Validation Error')
+  @Response<InternalServerError>(500, 'Internal Server Error')
+  @Post('/estimate-priority-fee')
+  async estimatePriorityFee(@Body() body: EstimatePriorityFeeBody): Promise<number | undefined> {
```

Comment:
Unless there is a really good reason to leverage the local fee market on a per account basis, I would update this to just be a `getPriorityFees` function at `/fees/priority` that returns a similar slow/average/fast response to `/gas/fees` (evm) leveraging `includeAllPriorityFeeLevels`. This provide us the same shape and "confirmation speeds" as the rest of the coinstacks via the global fee market.

Reply:
Very nice, I didn't see `includeAllPriorityFeeLevels` and I'm super stupid I thought one of accountKeys or transaction param was needed to get those fees!

Refactored as you want in [0574829](https://github.com/shapeshift/unchained/pull/1050/commits/05748299dc8e39bff8892107ee1ab536f3a4722b)

And added base fees in [2d662c9](https://github.com/shapeshift/unchained/pull/1050/commits/2d662c9ee3b59d4a7fb96236acaf9ba50f619baa)